bindings/python/crocoddyl/utils/__init__.py

```diff
@@ -579,8 +598,8 @@ def calc(self, data, x):
         data.a0[:] = pinocchio.getFrameAcceleration(self.state.pinocchio, data.pinocchio,
                                                     self.xref.frame).linear + pinocchio.utils.cross(self.vw, self.vv)
         if self.gains[0] != 0.:
-            data.a0[:] += np.asscalar(
-                self.gains[0]) * (data.pinocchio.oMf[self.xref.frame].translation - self.xref.oxf)
+            data.a0[:] += np.asscalar(self.gains[0]) * (
```

Comment:
I believe this change will produce a problem in the Python code format.

We are also checking the Python code format, please run `yafp -ri .` in the root folder. More explanations are in the wiki!